phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixDriver.java

```diff
@@ -182,7 +196,7 @@ public void onRemoval(RemovalNotification<ConnectionInfo, ConnectionQueryService
     
 
     @Override
-    public QueryServices getQueryServices() throws SQLException {
+    public QueryServices getQueryServices(final Properties info) throws SQLException {
```

Comment:
Adding extra argument won’t override the method

Reply:
> Adding extra argument won’t override the method

I find this getQueryServices only use in inner the project， only test and PhoenixDriver use.   client use by PhoenixDriver, do not oveeride this interface.